src/renderer.ts

```diff
@@ -370,10 +375,30 @@ const render = (salt: string, pillar: Pillar) => (element: BodyElement, key: num
 const renderAll = (salt: string) => (pillar: Pillar, elements: BodyElement[]): ReactNode[] =>
     elements.map(render(salt, pillar));
 
+const renderMedia = (salt: string) => (pillar: Pillar, elements: Image[]): ReactNode[] =>
```

Comment:
This is interesting. Is it possible that this is the first time we've needed to know what the `Design` is when rendering content for the body?

As a result, `renderAll` wasn't sufficient because it doesn't pass `Design` through?